### ISSUE TITLE
Illegal removal of voice channels

### ISSUE THREAD

**Issue by VoDACode** [LoggeL/AutoVoiceChannelBot#3]
When the user enters a voice channel (which creates other voice channels, let it be **A**), the bot moves the user to the newly created channel (let it be **B**). So far, everything is fine.
Next, when the user switches from the created channel (**B**) to some other (let it be **C**) - the created channel (**B**) is deleted. So far, everything is fine.
But when the user leaves that channel (**C**), it is also deleted by the bot.

**Comment by LoggeL** [comment]
Channel C was created by the bot? 
In which category is that channel hoisted?

**Comment by VoDACode** [comment]
1. No
2. To the same one specified in the "categoryName" parameter

**Comment by LoggeL** [comment]
Yeah the bot tries to clean up all channels in the given Category which are empty when left. 
It was meant to clean up zombie channels. Will think about reworking it in the future.
